app/views/admin/invoices/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1> Admin Dashboard </h1>
+
+<h2> Invoice ID: <%= @invoice.id %> </h2>
+<p> Status: <%= @invoice.status %> </p>
+<p> Created at: <%= @invoice.created_at.strftime("%A, %B %d, %Y") %> </p>
+<h3> Customer: <%= "#{@invoice.customer.first_name} #{@invoice.customer.last_name}" %> </h3>
+<br>
+<h2> Invoice Items </h2>
+
+<% @invoice.invoice_items.each do |invoice_item| %>
+  <h3> Item: <%= invoice_item.item.name%> </h3>
+  <p> Quantity: <%= invoice_item.quantity %> </p>
+  <p> Unit Price: $<%= '%.2f'% ((invoice_item.item.unit_price.to_f) / 100)  %> </p>
```

Comment:
Same thing in this section, possibly instantiating the invoice_items would clean this up, only concern is Law of Demeter, as we are reaching through three objects to get to item here.